src/plugins/intel_gna/src/frontend/scale_factor_calc.cpp

```diff
@@ -921,7 +922,7 @@ bool ScaleFactorCalculator::ScaleFactorPerLayerEltwise(InferenceEngine::EltwiseL
             return true;
         }
 
-        auto weightsReducer = calculateWeightsReducerFromDstStats(quantData->_dst_quant);
+        float weightsReducer = static_cast<float>(calculateWeightsReducerFromDstStats(quantData->_dst_quant));
         if (weightsReducer > initial_weights_reducer_val) {
```

Comment:
initial_weights_reducer_val is double and it was compared to double , but you changed it to float.
This is quite risky, due the fact precision is very important here. I would not change type from double to float.

If this is only to fix warning in line 927 I would change it like that.
```
            float newOutputScale = quantParams1->_dst_quant.GetScale() / weightsReducer;
//to
            float newOutputScale = quantParams1->_dst_quant.GetScale() / static_cast<float>(weightsReducer);

```